pkg/kn/commands/service/migrate.go

```diff
@@ -195,7 +195,7 @@ func NewServiceMigrateCommand(p *commands.KnParams) *cobra.Command {
 						if err != nil {
 							return err
 						}
-						fmt.Println("Migrate revision", color.CyanString(revision.Name), "successfully")
+						fmt.Println("Migrate revision", revision.Name, "successfully")
```

Comment:
Change to “Migrated revision”

Reply:
Fixed in https://github.com/knative/client/pull/406/commits/bb3fd28369fc262e8757bce812b525effe74fe3d thanks!